content/using/install.md

```diff
@@ -148,217 +116,133 @@ use |ota %disable or |ota ~sponsor %kids to reset it
+4. Click the "Download Arvo Keyfile" button. If this is grayed out, click on "Reset Networking Keys", then click on "Reset Networking Keys" on the following page, validate the transaction using your wallet, and then click "Send Trasaction". Once the transaction is complete, the "Download Arvo Keyfile" button should be available for you to press. You should receive a `.key` file that contains the secret needed to boot your planet. Hold onto this file.
 
-There are two supported ways of running an Urbit ship: using a cloud service, or running it locally on your own machine. Most users run things locally at first, but we recommend eventually using a cloud service for Urbit because it allows your ship to be accessed from anywhere on multiple devices. Hosting your ship in the cloud also allows it to always be online and ready for OTA updates. It's technically possible to run your Urbit ship on a home server, but ISPs often restrict this to business plans and opening up your home network to the internet can be unsafe if done improperly.
+### Run the boot command {#the-dojo}
 
-We have a guide for [hosting your ship on DigitalOcean](@/using/operations/hosting.md) which we've found works well, but any cloud hosting service should work.
+Navigate to your `urbit` directory.
 
-### Boot your planet
-
-You'll be booting your ship with the keyfile that you downloaded from Bridge.
-
-#### Step 1: Find the path to your keyfile
-
-Find the absolute path to the keyfile that you downloaded from Bridge. Copy it.
-
-#### Step 2: Run the boot command
-
-Enter your `urbit` directory.
-
-Once you're inside, run the command below, except with `sampel-palnet` replaced by the name of your
-Urbit identity, and `path/to/my-planet.key` replaced with the path to your keyfile:
-
-```sh
-./urbit -w sampel-palnet -k path/to/my-planet.key
-```
-
-Or, if you'd prefer to copy-paste your key, you can run:
+Open your keyfile (it is just a text file) and copy the key inside.  Paste it into the following command, with `sampel-palnet` replaced by the name of your
+planet.
 
 ```sh
 ./urbit -w sampel-palnet -G rAnDoMkEy
 ```
 
-Either command will create a directory called `sampel-palnet/` and begin building your ship. It may take a few minutes.
-
-When your ship is finished booting, you will see either the `~sampel-palnet:dojo>` or `~sampel-palnet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-X` to switch into Dojo.
-
-To shut down your ship, use `Ctrl-D`. To start your ship up again, run the following from your `urbit` directory:
+Or you can copy the keyfile into the current directory and run:
 
 ```sh
-./urbit sampel-palnet
+./urbit -w sampel-palnet -k ./my-planet.key
 ```
 
-Note that `sampel-palnet/` is the path of a folder, which we just created in your `urbit` directory. This folder is called your ship's **pier**.
+Either command will create a directory called `sampel-palnet/` and begin building your planet. It may take a few minutes.
```

Comment:
```suggestion
Either command will create a directory called `sampel-palnet/` and begin booting your planet. It may take a few minutes.
```

The language we use elsewhere is booting, not building, so I'd recommend we stick with it.